staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/admission.go

```diff
@@ -196,14 +204,52 @@ func (a *GenericAdmissionWebhook) Admit(attr admission.Attributes) error {
 	hooks := hookConfig.ExternalAdmissionHooks
 	ctx := context.TODO()
 
-	errCh := make(chan error, len(hooks))
+	hookIndexCh := make(chan int, len(hooks))
 	wg := sync.WaitGroup{}
 	wg.Add(len(hooks))
 	for i := range hooks {
+		go func(hook *v1alpha1.ExternalAdmissionHook, i int) {
```

Comment:
If you pass `i` you don't need to pass `hook`?

Reply:
Is there a reason to call these in parallel? Seems like a lot of code for possibly small benefit?

Reply:
Removed.